### PR TITLE
SOEN-163 Added cross for clearer visual in modal

### PR DESCRIPTION
**Pull request by SergioAAI**
Co-authored-by: raisa61 <raisa61@users.noreply.github.com>

# Description

Implements/fixes [SOEN-163](https://soen390team8.atlassian.net/browse/SOEN-163)

- Added a Cross to indicate that a status is not reviewed

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Unit Tests

# Screenshots

Checkmark

### CODE REVIEW

File: frontend/components/doctor/patient-info-modal-swiper.tsx [hunk 1]
```diff
@@ -68,15 +69,15 @@ const PatientInfoModalSwiper = ({ patient, onMutate }: AppProps) => {
...
+                                <HStack>
                                     <Box>
                                         <CheckMark
                                             isColored={statusInfo.isReviewed}
                                             color="#1F9D00"
                                             onClicking={() => {
                                                 reviewHandler(statusInfo.statusId);
                                             }}
-                                            isUnfillable={"false"}
+                                            isUncheckable={"false"}
```
Comment (NicholasYip) [inline frontend/components/doctor/patient-info-modal-swiper.tsx:80]: should this be a string or a boolean? I feel like a boolean would be better

**Review by NicholasYip — APPROVED** [review]
Small question, rest looks good to me

File: frontend/components/UI/checkmark.tsx [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
         } else {
             return;
         }
     };
 
     return (
         <Box onClick={clickHandler}>
-            <Check fill={isFilled ? color : "black"} />
+            {isChecked && <Check fill={color} />}
+            {!isChecked && <Cross fill={"#E53945"} />}
```
Comment (amqkv) [inline frontend/components/UI/checkmark.tsx:33]: Just for a more intuitive UI, can we replace the cross with like a checkbox or something? And make the cursor become a pointer when it hovers over it so the user knows that it's clickable?

File: frontend/components/doctor/patient-info-modal-swiper.tsx (hunk 1 above)
Comment (SergioAAI) [reply frontend/components/doctor/patient-info-modal-swiper.tsx:80]: Stayed like that when debugging a weird interaction , it should be a simple boolean

File: frontend/components/UI/checkmark.tsx (hunk 2 above)
Comment (SergioAAI) [reply frontend/components/UI/checkmark.tsx:33]: Ill try to change it to a checkbox

**Review by stevenvuong00 — APPROVED** [review]
nice work as always!

**Review by KelvinTrang — APPROVED** [review]
good job!

**Pull request merged**